site/blog/2019/sep/19/helping-first-time-conference-attendees-with-welcome-wagon.rst

```diff
@@ -0,0 +1,87 @@
+Helping First-Time Conference Attendees with the Welcome Wagon
+==============================================================
+
+You walk into the room and you're wearing the *wrong* thing.
+Everyone else is dressed casually,
+but you are not.
+You feel silly and find the quickest excuse to leave the situation.
+This is an example of a common situation:
+not knowing what is expected of you when you enter a new place.
+
+It's stressful to enter a new group without knowing what to expect.
+It breeds anxiety and self-doubt,
+and makes it much more likely you'll find excuses not to join.
+
+At Write the Docs,
+we're working to reduce the possibility of this happening.
```

Comment:
If you decide to go with the friend in high school analogy, you can use it here too:  
> We want to be that friend you can call to find out what to wear, or bring, or expect of your experience.

Reply:
I think the idea of having a friend at the event ahead of time is an interesting analogy. We can likely do it without needing the high-school reference. I'll think a bit more about it.